### DISCUSSION TITLE
Spinner score isn't included in scores set in stable

### DISCUSSION THREAD

**Discussion by plaguemask** [ppy/osu#17328] [Q&A]
When looking at a score that was set in stable, Spinner Spin and Spinner Bonus are always 0, and score from spinners isn't factored into the total score. The only way to see a play's true score is to watch the replay.

Leaderboard shown not including spinner score:
![osu_2022-03-18_12-43-55](https://user-images.githubusercontent.com/66150801/159055672-5cb38d4f-454e-4d8a-9ea5-56094660cccd.jpg)

True score as shown after replay:
![osu_2022-03-18_12-44-58](https://user-images.githubusercontent.com/66150801/159056847-94467b52-f98f-4e7b-a8b9-50a386feb39d.jpg)

**Comment by bdach** [comment]
I don't know what the answer is here. The amount of spinner bonus points isn't available easily from a replay file without simulating the whole replay I think. So there is no good way to pull that data out from before the replay is played back.

Maybe this will cease to be a problem depending on results of the ongoing [scoring survey](https://osu.ppy.sh/home/news/2022-03-07-community-meetings-recap). If the community says that they don't want existing scores to be on the same leaderboard as new ones then maybe the old scores could remain untouched as is.

**Reply by peppy** [reply]
Another potential direction is that we recalculate scores which *do* have replays to get some rough numbers to fill these statistics. Obviously a lot more work and probably opens the door to new issues (scores will match lazer expectations but potentially not stable anymore), but it's one other avenue.
